content/en/blog/2025/infernece-extension-support/index.md

```diff
@@ -0,0 +1,155 @@
+
+This resource exclusivity means that simple round-robin or least-connection algorithms can create severe imbalances. Sending requests to a server that's already processing a complex inference task doesn't just add latency, it can cause resource contention that impacts performance for all queued requests.
+
+### **Stateful Considerations and Memory Management**
+
+AI models often maintain in-memory caches that significantly impact performance. KV caches store intermediate attention calculations for previously processed tokens, serving as the primary consumer of GPU memory during generation and often becoming the most common bottleneck. When KV cache utilization approaches limits, performance degrades dramatically, making cache-aware routing essential.
+
+Additionally, many modern AI deployments use fine-tuned adapters like [LoRA](https://arxiv.org/abs/2106.09685) (Low-Rank Adaptation) to customize model behavior for specific users, organizations, or use cases. These adapters consume GPU memory and loading time when switched. A model server that already has the required LoRA adapter loaded can process requests immediately, while servers without the adapter face expensive loading overhead that can take seconds to complete.
+
+### **Queue Dynamics and Criticality**
+
+AI inference workloads also introduce the concept of request criticality that's less common in traditional services. Real-time interactive applications (like chatbots or live content generation) require low latency and should be prioritized, while batch processing jobs or experimental workloads can tolerate higher latency or even be dropped during system overload.
+
+Traditional load balancers lack the context to make these criticality-based decisions. They can't distinguish between a time-sensitive customer support query and a background batch job, leading to suboptimal resource allocation during peak demand periods.
+
+This is where inference-aware routing becomes critical. Instead of treating all backends as equivalent black boxes, we need routing decisions that understand the current state and capabilities of each model server, including their queue depth, memory utilization, loaded adapters, and ability to handle requests of different criticality levels.
+
+## **Gateway API Inference Extension: A Kubernetes-Native Solution**
+
+The [Kubernetes Gateway API Inference Extension](https://gateway-api-inference-extension.sigs.k8s.io/guides/#__tabbed_3_2) has introduced solutions to these challenges, building on the proven foundation of Kubernetes Gateway API while adding AI-specific intelligence. Rather than requiring organizations to patch together custom solutions or abandon their existing Kubernetes infrastructure, the extension provides a standardized, vendor-neutral approach to intelligent AI traffic management.
```

Comment:
when introducing the concept, probably best to link to the root (https://gateway-api-inference-extension.sigs.k8s.io/), not the Istio tab in the docs?